ivy/functional/frontends/paddle/vision/transforms.py

```diff
@@ -33,3 +33,11 @@ def vflip(img, data_format="CHW"):
     elif data_format.lower() == "hwc":
         axis = -3
     return ivy.flip(img, axis=axis)
+
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": ("float16", "float32", "int16", "int8", "uint8")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def crop(img, top, left, height, width):
+    array = ivy.array(img)
```

Comment:
This conversion should be handled in `@to_ivy_arrays_and_back`, so I think calling `ivy.array` here is redundant.